src/aac/plugins/gen_plant_uml/gen_plant_uml_impl.py

```diff
@@ -4,8 +4,6 @@
 from aac import parser, util
 from aac.validator import validation
 
-plugin_version = "0.0.2"
```

Comment:
"Removed 1st party plugin versions since they'll always be versioned by the AaC package"

1st party plugins don't need versions since their version is inherently linked to the AaC package.